integrations/huggingface-transformers/tutorials/sparse-transfer-learning-bert.md

```diff
@@ -0,0 +1,343 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Sparse Transfer Learning
+
+This page explains how to fine-tune a pre-sparsified BERT model onto a downstream dataset with SparseML.
+
+## Overview
+
+Sparse Transfer is quite similiar to the typical NLP transfer learning, where we fine-tune a checkpoint pretrained on a large dataset like Wikipedia BookCorpus onto a smaller downstream dataset and task. However, with Sparse Transfer Learning, we simply start the fine-tuning process from a pre-sparsified model and maintain sparsity while the training process occurs.
+
+[SparseZoo](https://sparsezoo.neuralmagic.com/?domain=nlp&sub_domain=masked_language_modeling&page=1), Neural Magic's open-source Model Zoo, contains pre-sparsified checkpoints of common NLP models like BERT-base, BERT-large, and RoBERTa. These models can be used as the starting checkpoint for the sparse transfer learning workflow.
+
+## Installation
+
+Install via `pip`:
+
+```
+pip install sparseml[torch]
+```
+
+## Sparse Transfer Learning onto SST2
+
+Let's try a simple example of fine-tuning a pre-sparsified model onto the SST dataset. SST2 is a sentiment analysis
```

Comment:
SST2 is a sentiment analysis dataset with pairs of sentences and binary labels representing positive or negative sentiment.